src/python/pants_release/generate_github_workflows.py

```diff
@@ -84,6 +84,8 @@ class Platform(Enum):
 
 GITHUB_HOSTED = {Platform.LINUX_X86_64, Platform.MACOS12_X86_64}
 SELF_HOSTED = {Platform.LINUX_ARM64, Platform.MACOS10_15_X86_64, Platform.MACOS11_ARM64}
+# We control these runners, so we preinstall and expose python on them.
```

Comment:
Technically these are the same as SELF_HOSTED, but it made sense to create a distinction.